RetailCoder.VBE/UI/ToDoItems/ToDoExplorerDockablePresenter.cs

```diff
@@ -45,8 +47,9 @@ public override void Show()
             try
             {
                 Cursor.Current = Cursors.WaitCursor;
-                Control.TodoItems = await GetItems();
+                _view.TodoItems = await GetItems();
             }
+            //wtf? why are we catching everything without even so much as leaving a comment as to why?
             catch { }
```

Comment:
Looks like it was @Hosch250 [GitBlame](https://github.com/rubberduck-vba/Rubberduck/blame/e814efb08e5644cd378efc7e62efedae8359a48d/RetailCoder.VBE/UI/ToDoItems/ToDoExplorerDockablePresenter.cs). Hey buddy, care to explain what we're doing here? Looks like it was somehow related to #577.